home/dot_agents/skills/working-in-systalyze-worktrees/scripts/resolve_premerge_stack.py

```
@@ -0,0 +1,794 @@
+            command="git",
+            sshCommandQueryFailed=True,
+            returnCode=result.returncode,
+            stdoutSha256="sha256:"
+            + hashlib.sha256(result.stdout.encode("utf-8")).hexdigest(),
+            stderrSha256="sha256:"
+            + hashlib.sha256(result.stderr.encode("utf-8")).hexdigest(),
+        )
+    return result.stdout.rstrip("\n")
+
+
+def noninteractive_ssh_command(
```

Ruff (0.16.3)

[warning] Too many branches (19 > 12)

(PLR0912)